tools/ramclustr/ramclustr.xml

```diff
@@ -0,0 +1,62 @@
+<tool id="ramclustr" name="RAMClustR" version="@TOOL_VERSION@+galaxy1">
+    <macros>
+        <import>ramclustr_macros.xml</import>
+    </macros>
+
+    <requirements>
+        <container type="docker">recetox/ramclustr:1.1.0-recetox0</container>
+    </requirements>
+
+    <inputs>
+        <param name="input_xcms" label="input_xcms" type="data" format="RData" help=": containing grouped feature data for clustering by ramclustR" />
+        <expand macro="parameters" />
+    </inputs>
+
+    <command detect_errors="aggressive"><![CDATA[
+    
+        Rscript 
+            -e 'obj <- load("$input_xcms")'
+            -e 'x <- RAMClustR::ramclustR(
+                xcmsObj = xdata,
+                #if $params.MStag 
+                    MStag = "$params.MStag",
+                #end if
+                #if $params.idMSMStag 
+                    idMSMStag = "$params.idMSMStag",
+                #end if
+                taglocation = "$params.taglocation",
+                sr = $params.sr,
+                #if $params.st 
+                    st = $params.st,
+                #end if
+                #if $params.maxt 
+                    maxt = $params.maxt,
+                #end if
+                deepSplit = $params.deepSplit,
+                blocksize = $params.blocksize,
+                mult = $params.mult,
+                hmax = $params.hmax,
+                collapse = $params.collapse,
+                usePheno = $params.usePheno,
+                mspout = $params.mspout,
+                qc.inj.range = $params.qc_inj_range,
+                normalize = "$params.normalize",
+                minModuleSize = $params.minModuleSize,
+                linkage = $params.linkage,
+                mzdec = $params.mzdec,
+                cor.method = "$params.cor_method",
+                rt.only.low.n = $params.rt_only_low_n,
+                replace.zeros = $params.replace_zeros
+                )'
+            -e 'save(x, file = "$result")'
+    ]]>
+    </command>
+
+    <outputs>
+        <data name="result" format="RData" />
+    </outputs>
+
```

Comment:
```suggestion
<tests>
  <test>
    <param ... />
  </test>
</tests>
```

Please add some tests - to retrieve the reference outputs just run it once on your machine on some open-sourced dataset. One such mzml dataset is in the `aplcms` directory.

Reply:
There is yet no way to run the tool completely, I'd like to include it in our version of Galaxy for further testing.

Reply:
@hechth How does a tool that cannot be run yet benefit from being available on UMSA Galaxy?

Reply:
I wired in the parameters and can't yet tell if everything runs smoothly as the parameters have to be tuned for the data, which is something where I'd like @ElliottJP to have a look at.

If no non completely tested tool should be in the Galaxy I'll test these things differently.

Reply:
I am fine with making this available on UMSA, we are in a development phase and everything there should be considered beta at best. However, I would also like to see a test if possible, since that documents how to use the tool and makes it much more accessible and easier to reason about I think. 

If you are saying test is not helpful at the moment, that is fine with me.